test/powershell/Modules/ThreadJob/ThreadJob.Tests.ps1

```diff
@@ -93,24 +93,30 @@ Describe 'Basic ThreadJob Tests' -Tags 'CI' {
 
         $job = Start-ThreadJob -ScriptBlock { "Goodbye" } -InitializationScript { "Hello" }
         $results = $job | Receive-Job -Wait
-        $results[0] | Should -Be "Hello"
-        $results[1] | Should -Be "Goodbye"
+        $results | Should -BeExactly (
+            "Hello",
+            "Goodbye"
+        )
```

Comment:
You can drop the commas on all of these if you use `@()`:
```suggestion
        $results | Should -BeExactly @(
            "Hello"
            "Goodbye"
        )
```

Reply:
It makes the test less readable. I can not accept this.